Cargo.toml

```diff
@@ -22,7 +22,7 @@ libc = "0.2"
 url = "2.2.2"
 lazy_static = "1.4"
 encoding = "0.2"
-
+wasi = "0.11.0+wasi-snapshot-preview1"
```

Comment:
please select your need function to `src/event_loop/wasi_fs.rs` , and remove `wasi` use.